#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raise a nameError message."""
    raise NameError(message)
